### PR TITLE
Updated-Validation of Routing and Dispatch

### PR DESCRIPTION
**Pull request by sadaf510**
I have put validations and placeholder updates for Orders, trips, routes, cross-border, and documents. 
I have also disabled the save buttons for these modules.

There were functionality errors in order-  I could not save or update the order once the consignor and consignee got added. This issue was discussed with @Daljit-fleetHawks and once or gets merged this will be resolved.

There was also a functionality issue in trips, after adding a new plan trip, the save and update button remains disabled. This issue was discussed with @FH-sandeepkaur1 and soon to be resolved.

Rest Route, Cross-border, and Document functionalities are working.  Kindly let me know still any changes to do.

### CODE REVIEW

**Comment by kunalfleethawks** [comment]
@Hardeep8587 or @FH-sandeepkaur1  Please review and approve

File: src/app/pages/dispatch/cross-border/aci-documents/new-aci-manifest/new-aci-manifest.component.html [hunk 1]
```diff
@@ -123,7 +123,7 @@
                           <div>
 
                             <div *ngIf="Etime1.errors.required">
-                             Select required Estimated Arrival Time.
+                             Estimated Arrival Time is required.
```
Comment (FH-sandeepkaur1) [inline src/app/pages/dispatch/cross-border/aci-documents/new-aci-manifest/new-aci-manifest.component.html]: Please take care of lower and upper case in validations and placeholders. First letter should be capital in the sentence followed by small characters in rest

File: src/app/pages/dispatch/cross-border/aci-documents/new-aci-manifest/new-aci-manifest.component.html [hunk 2]
```diff
@@ -39,21 +39,21 @@
...
                             <input type="text" class="form-control" name="tripNumber"
-                              placeholder="8 or 25 digits , eg. TEST0001 " [(ngModel)]="tripNumber" #TripNum1="ngModel" required pattern="^[A-Z]{4}[A-Z0-9]{4,21}$" >
+                              placeholder="8 or 25 digits , eg. TEST0001 " [(ngModel)]="tripNumber" #TripNum1="ngModel" required pattern="^[A-Z0-9]{4,21}$" >
                               <div *ngIf="TripNum1.invalid && (TripNum1.dirty || TripNum1.touched)" class="text-danger">
                                 <div *ngIf="TripNum1.errors.required">
                                   Trip Number is required.
                                 </div>
                                 <div *ngIf="TripNum1.errors.pattern">
-                                  Trip Number must contain alpha-numeric with no spaces or special characters, letters must be upper case.
+                                  Trip Number must be of 4-21 upper case letters and numbers with no spaces or special characters.
```
Comment (FH-sandeepkaur1) [inline src/app/pages/dispatch/cross-border/aci-documents/new-aci-manifest/new-aci-manifest.component.html]: The pattern of messages and placeholders should be same through out the solution. We are using the  term alphanumeric where letters and numbers are required. Please verify it from other modules and make changes accordingly

File: src/app/pages/dispatch/cross-border/aci-documents/new-aci-manifest/new-aci-manifest.component.html [hunk 3]
```diff
@@ -398,7 +398,7 @@
 
 
                           <div class="col-lg-10">
-                            <label class="control-label font-weight-bold text-lg-right pt-2">Issuing Province<span
+                            <label class="control-label font-weight-bold text-lg-right pt-2">Issuing State or Province<span
```
Comment (Hardeep8587) [inline src/app/pages/dispatch/cross-border/aci-documents/new-aci-manifest/new-aci-manifest.component.html]: @sadaf510 make it State/Province, everywhere

**Review by FH-sandeepkaur1 — COMMENTED** [review]
Please take care of the format and pattern of the messages and placeholders in all the files

File: src/app/pages/dispatch/cross-border/aci-documents/new-aci-manifest/new-aci-manifest.component.html (hunk 1 above)
Comment (sadaf510) [reply src/app/pages/dispatch/cross-border/aci-documents/new-aci-manifest/new-aci-manifest.component.html]: done

File: src/app/pages/dispatch/cross-border/aci-documents/new-aci-manifest/new-aci-manifest.component.html (hunk 3 above)
Comment (sadaf510) [reply src/app/pages/dispatch/cross-border/aci-documents/new-aci-manifest/new-aci-manifest.component.html]: done

File: src/app/pages/dispatch/cross-border/aci-documents/new-aci-manifest/new-aci-manifest.component.html (hunk 2 above)
Comment (sadaf510) [reply src/app/pages/dispatch/cross-border/aci-documents/new-aci-manifest/new-aci-manifest.component.html]: done

**Review by kunalfleethawks — APPROVED** [review]

**Pull request merged**